### DISCUSSION TITLE
Developing an app that supports documented 'Best Practices'

### DISCUSSION THREAD

**Discussion by mattmartin26** [Cath-Strategic-Tech/Refugees-Support#37] [General]
The IRCC has a page of a few whitepapers related to sponsorship best practices.  I think we should probably review the whitepapers, in particular, the third, to look for how this application can best support the practices the government is recommending to SAHs.

[Best Practices](https://www.rstp.ca/en/resources/best-practices/)
[White Paper: Sponsoring Agreement Holders working with Constituent Groups and Co-sponsors](https://www.rstp.ca/wp-content/uploads/2014/03/Best-Practice-Vol-4-Knowledge-Sharing-Document.pdf)

@prescod @GeorgeM2021

**Comment by GeorgeM2021** [comment]
Hi Matt,

 

Apologies but I will not be able to join the meeting today.   I have a high priority work meeting that was injected into my schedule and I must attend it.

 

Thanks for sending the info.

 

From: Matthew Martin ***@***.***> 
Sent: Friday, December 3, 2021 8:45 AM
To: Cath-Strategic-Tech/Refugees-Support ***@***.***>
Cc: GeorgeM2021 ***@***.***>; Mention ***@***.***>
Subject: [Cath-Strategic-Tech/Refugees-Support] Developing an app that supports documented 'Best Practices' (Discussion #37)

 

The IRCC has a page of a few whitepapers related to sponsorship best practices. I think we should probably review the whitepapers, in particular, the third, to look for how this application can best support the practices the government is recommending to SAHs.

Best Practices <https://www.rstp.ca/en/resources/best-practices/> 
White Paper: Sponsoring Agreement Holders working with Constituent Groups and Co-sponsors <https://www.rstp.ca/wp-content/uploads/2014/03/Best-Practice-Vol-4-Knowledge-Sharing-Document.pdf> 

@prescod <https://github.com/prescod>  @GeorgeM2021 <https://github.com/GeorgeM2021> 

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/Cath-Strategic-Tech/Refugees-Support/discussions/37> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AWVR67KAPNYCBTXTNU5P573UPDXX5ANCNFSM5JKDUGWQ> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .   <https://github.com/notifications/beacon/AWVR67N2YYDPOKP2PNOT3KDUPDXX5A5CNFSM5JKDUGW2YY3PNVWWK3TUL52HS4DFVJCGS43DOVZXG2LPN2VGG33NNVSW45C7NFSM4ABYZEQQ.gif>
